test/fixtures/output/kafka/mirror/resume-mirror.golden

```diff
@@ -0,0 +1,5 @@
+  Mirror Topic Name | Partition | Partition Mirror Lag | Last Source Fetch Offset | Error Message  | Error Code  
```

Comment:
How about "Topic Name" instead of "Mirror Topic Name"? The command is called `kafka mirror resume` so the user knows this is a mirror topic.

Reply:
Let's save that change for v4, so that the human and serialized labels stay consistent in the meantime.